core/src/main/scala/org/http4s/util/SSLContextFactory.scala

```diff
@@ -0,0 +1,67 @@
+package org.http4s.util
+
+import java.io.ByteArrayInputStream
+import java.security.cert.{CertificateFactory, X509Certificate}
+
+import javax.net.ssl.SSLSession
+
+import scala.util.Try
+
+/**
+  * Based on SSLContextFactory from jetty.
+  */
+object SSLContextFactory {
```

Comment:
I wonder about making this internal API.  As we think about 1.0, everything we add becomes more permanent.  I think `util` is generally a sign that it belongs in another library or that it belongs in the internal package as `protected[http4s]` so we aren't on the hook for supporting it.

Reply:
It was also part of "core" as I didn't find any "util" package in server repo. We can move this to blaze-server repo if this doesn't have to be in "util".